### ISSUE TITLE
how to  detach agentmain pid？

### ISSUE THREAD

**Issue by cor0ps** [raphw/byte-buddy#1161]
maybe  virtualMachine.detach(); can  remove   attach .

but  I  use  byteBuddyAgent.attach   and   how can  implements   detach function?

**Comment by raphw** [comment]
Byte Buddy detaches implicitly within the method. Detachment only affects the socket connection being used go send the sgent.
